modules/ROOT/pages/salesforce/salesforce-connector-100.adoc

```diff
@@ -0,0 +1,204 @@
+
+https://www.mulesoft.com/exchange/?search=salesforce&type=template[Anypoint Exchange templates] provide starting points for Anypoint Studio projects such as:
+
+* Account Broadcast template
++
+https://www.mulesoft.com/exchange/org.mule.templates/template-sfdc2sfdc.wday.sap.db-account-broadcast/[Salesforce to Salesforce, Workday, SAP, and Database]
+* Aggregation
++
+https://www.mulesoft.com/exchange/org.mule.templates/template-sfdc2sfdc-contact-aggregation/[contacts],
+https://www.mulesoft.com/exchange/org.mule.templates/template-sfdc2sfdc-opportunity-aggregation/[opportunity],
+https://www.mulesoft.com/exchange/org.mule.templates/template-sfdc2sfdc-user-aggregation/[user]
+* Bidirectional Sync template
++
+** Between Salesforce organizations
++
+https://www.mulesoft.com/exchange/org.mule.templates/template-sfdc2sfdc-account-bidirectional-sync/[accounts],
+https://www.mulesoft.com/exchange/org.mule.templates/template-sfdc2sfdc-contact-bidirectional-sync/[contacts],
+https://www.mulesoft.com/exchange/org.mule.templates/template-sfdc2sfdc-casecustomobject-bidirectional-sync/[custom objects],
+https://www.mulesoft.com/exchange/org.mule.templates/template-sfdc2sfdc-opportunity-bidirectional-sync/[opportunities],
+https://www.mulesoft.com/exchange/org.mule.templates/template-sfdc2sfdc-user-bidirectional-sync/[users]
++
+** Other sources
++
+https://www.mulesoft.com/exchange/org.mule.templates/template-sfdc2db-account-bidirectional-sync/[database],
+https://www.mulesoft.com/exchange/org.mule.templates/template-sfdc2msdyn-account-bidirectional-sync/[Microsoft Dynamics CRM],
+https://www.mulesoft.com/exchange/org.mule.templates/template-sfdc2sieb-account-bidirectional-sync/[Siebel]
++
+* Migration
++
+https://www.mulesoft.com/exchange/org.mule.templates/template-sfdc2sap-account-migration/[accounts],
+https://www.mulesoft.com/exchange/org.mule.templates/template-sap2sfdc-contact-migration/[contacts],
+https://www.mulesoft.com/exchange/org.mule.templates/template-sfdc2sfdc-opportunity-migration/[opportunities]
+
+=== Salesforce Connector Examples in Exchange
+
+https://www.mulesoft.com/exchange/?search=salesforce&type=example[Anypoint Exchange examples] enable you to create complete Anypoint Studio projects such as:
+
+* Batch processing
++
+https://www.mulesoft.com/exchange/org.mule.examples/salesforce-to-MySQL-DB-using-Batch-Processing/[database] and
+https://www.mulesoft.com/exchange/org.mule.examples/import-leads-into-salesforce/[import leads]
++
+* https://www.mulesoft.com/exchange/org.mule.examples/cache-scope-with-salesforce-contacts/[Contacts]
+* https://www.mulesoft.com/exchange/org.mule.examples/salesforce-data-retrieval/[Data retrieval]
+* https://www.mulesoft.com/exchange/org.mule.examples/dataweave-with-flowreflookup/[DataWeave]
+
+== Next
+
+After you have met the prerequisites and experimented with templates and examples, you are ready to create an app with xref:salesforce/salesforce-connector-design-center.adoc[Design Center] or
```

Comment:
present tense
After you have met the prerequisites and experimented with > After you meet the prerequisites and experiment with